prov/rxm/src/rxm_ep.c

```diff
@@ -1623,7 +1623,7 @@ static int rxm_ep_bind(struct fid *ep_fid, struct fid *bfid, uint64_t flags)
 
 	case FI_CLASS_CQ:
 		rxm_cq = container_of(bfid, struct rxm_cq, util_cq.cq_fid.fid);
-		if (rxm_ep->util_coll_ep && rxm_cq->util_coll_cq) {
+		if (rxm_ep->util_coll_ep) {
```

Comment:
need to add FI_CLASS_PEER_CQ to the `case` clause.

Reply:
Do you expect that somebody externally can bind peer_cq to rxm provider?
Or do we just want to reject such a case as rxm_ep does not support peer_cq so far?
```
	case FI_CLASS_PEER_CQ:
		return -FI_ENOSYS;
		break;
```

Reply:
My bad. This is not needed here. I was cross reading this and the util_coll additions.